react-app/src/native/components/register/ChooseLeague.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState} from 'react'
```

Comment:
This should essentially be a copy of /components/common/TeamPicker.js just for leagues instead of teams. I honestly thought I already had a generic league picker, it actually basically already exists in ScoreReporter.js on line 208

Reply:
PS i added comments to this file so if/when you move it to the common folder it can be refactored with recommendations.